airbyte-integrations/connectors/source-facebook-marketing/Dockerfile

```
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.9
+LABEL io.airbyte.version=0.2.9=10
```

This is a typo, right?

```suggestion
LABEL io.airbyte.version=0.2.10
```